File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.go
func findNumbers(nums []int) int {
    result := 0;

    for _, num := range nums {
        if num == 100000 || (num < 10000 && num > 999) || (num < 100 && num > 9) {
            result++;
        }
    }

    return result;
}

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int result = 0;

        for (int num : nums) {
            if (num == 100000 || (num < 10000 && num > 999) || (num < 100 && num > 9)) result++;
        }

        return result;
    }
}